libraries/aws_iam_root_user.rb

```diff
@@ -0,0 +1,24 @@
+# author: Miles Tjandrawidjaja
+class AwsIamRootUser < Inspec.resource(1)
+  name 'aws_iam_root_user'
+  desc 'Verifies settings for AWS root account'
+  example "
+    describe aws_iam_root_user do
+      its('access_key_count') { should eq 0 }
+    end
+  "
+
+  def initialize(conn = AWSConnection.new)
+    @client = conn.iam_client
+  end
+
+  def access_key_count
+    summary_account['AccountAccessKeysPresent']
```

Comment:
Nice investigation. For some reason I thought ruby hashes were indifferent to string vs symbol and treated them as equal. good to know I'm wrong